hack/deploy-argocd.sh

```diff
@@ -3,15 +3,14 @@
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"/..
 
 main() {
+    local argocd=${1:?"Kustomization directory for ArgoCD subscription not provided"}
     verify_permissions || exit $?
-    create_subscription
+    create_subscription $argocd
     wait_for_route
     switch_route_to_reencrypt
     grant_admin_role_to_all_authenticated_users
     mark_pending_pvc_as_healty
-    add_role_binding
```

Comment:
FYI, this was removed because the kustomization directory used to call `create_subscription` includes the necessary roles and role bindings already.